app/models/concerns/listable.rb

```diff
@@ -15,7 +16,7 @@ def next
     end
 
     def most_recent
-      past.load.first
+      past.includes(:sponsors).load.first
```

Comment:
This can be replaced with `joins`?

Reply:
I fixed this based on the warnings I got from [bullet](https://github.com/flyerhzm/bullet) - I'm not sure tbh but this seems to resolve the error.